src/routers/transactions/products/index.ts

```diff
@@ -133,7 +133,15 @@ router.get('/:type', async (req, res) => {
  */
 router.get('/:type/download', async (req, res) => {
   const result = await ProdTransactionsService.getProductTransactionsCsv(req.params.type);
+  
+  const date = new Date();
+  const dateString = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`; 
+  const filename = `collections-${dateString}.csv`;
+  
+  res.setHeader('Content-disposition', `attachment; filename=${filename}`);
+  res.set('Content-Type', 'text/csv');
   res.status(StatusCode.OK).send(result);
+
```

Comment:
unnecessary line

Reply:
talked to @jeinglis about this and we thought it would be good to name it collections instead so it is consistent with the "collect" tab in agritrader

Reply:
also i had issues with using this way of downloading a csv from agriboard so the name of the file is generated in my agriboard pull request instead